app/src/main/java/com/keylesspalace/tusky/components/account/media/AccountMediaRemoteMediator.kt

```diff
@@ -58,7 +60,7 @@ class AccountMediaRemoteMediator(
             }
 
             val attachments = statuses.flatMap { status ->
-                AttachmentViewData.list(status)
+                AttachmentViewData.list(status, accountManager.activeAccount?.alwaysShowSensitiveMedia ?: false)
```

Comment:
The active account can change over the lifecycle of this object (there's a race condition between the active account changing and the object being destroyed).

The safe thing to do is add an `activeAccount` property, like

```
val activeAccount = accountManager.activeAccount!!
```

and then use it here.